sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpers.java

```diff
@@ -411,6 +411,21 @@ public static TableReference parseTableSpec(String tableSpec) {
     return ref.setDatasetId(match.group("DATASET")).setTableId(match.group("TABLE"));
   }
 
+  public static TableReference parseTableUrn(String tableUrn) {
```

Comment:
I couldn't find any usages for this here.

Reply:
Yes - this is used in the followon PR. Added a unit test in this PR.